packages/scoped-elements/src/ScopedElementsMixin.js

```diff
@@ -1,4 +1,4 @@
-import './loadPolyfill.js';
+import '@webcomponents/scoped-custom-element-registry';
```

Comment:
does this now include the logic to not double load if it's executed 2 times from e.g. nested node_modules? 🤔

Reply:
Yes, it includes all that logic plus all the fixes we send last month, even the logic to "upgrade" CE created before applying the polyfill

Reply:
sweet 💪